command_line/mono_pkg.yaml

```diff
@@ -9,9 +9,9 @@ os:
 
 stages:
   - analyze:
```

Comment:
In the PR I did, I renamed the stage to `analysis`, as I found it confusing to have a stage and a command have the exact same name. WDYT @kevmoo ?